ios/utils/UINavigationBar+RNSUtility.mm

```diff
@@ -41,4 +41,44 @@ - (nullable UIView *)rnscreens_findBackButtonWrapperView
   return nil;
 }
 
+- (NSDirectionalEdgeInsets)rnscreens_sumTitleControlDirectionalMargins
+{
+  NSDirectionalEdgeInsets totalMargins = NSDirectionalEdgeInsetsMake(0, 0, 0, 0);
+  UIView *titleControl = RNScreensFindTitleControlInView(self);
+
+  if (!titleControl) {
+    NSLog(@"[RNScreens] _UINavigationBarTitleControl not found under the UINavigationBar hierarchy");
+    return totalMargins;
+  }
+
+  UIView *currentView = titleControl.superview;
+  while (currentView && currentView != self) {
+    NSDirectionalEdgeInsets margins = currentView.directionalLayoutMargins;
+    totalMargins.top += margins.top;
+    totalMargins.leading += margins.leading;
+    totalMargins.bottom += margins.bottom;
+    totalMargins.trailing += margins.trailing;
+    currentView = currentView.superview;
+  }
```

Comment:
This loop also does not include insets of `UINavigationBar` itself, is it intentional?